PowerEditor/installer/nppSetup.nsi

```diff
@@ -270,64 +270,33 @@ Var nppSubStr
 
 ${MementoSection} "Context Menu Entry" explorerContextMenu
 
-	${If} $WinVer == "11"
-	
-		; Clean up the hack of v8.5 installer
-		ReadRegStr $muiVerbStr HKLM "SOFTWARE\Classes\*\shell\pintohome" MUIVerb
-		${StrStr} $nppSubStr $muiVerbStr "Notepad++"
-		; Make sure there's an entry, and the entry belong to Notepad++ before deleting it
-		${If} $muiVerbStr != ""
-			${AndIf} $nppSubStr != ""  ; it contains "Notepad++"
-				DeleteRegKey HKLM "SOFTWARE\Classes\*\shell\pintohome"
-		${EndIf}

```

Comment:
Hold, the above code for clean up v8.5 hijacking is not in installer?
It should add back in the beginning of this block.

Reply:
Ha! Isn't it better to clean up via Installer?

Reply:
That is in the installer:

```
LRESULT CleanupHack()
{
    wstring keyName = L"SOFTWARE\\Classes\\*\\shell\\pintohome";
    wstring valueName = L"MUIVerb";

    LRESULT result = 0;
    bool found = false;

    HKEY hkey;
    HRESULT hResult = RegOpenKeyEx(HKEY_LOCAL_MACHINE, keyName.c_str(), 0, KEY_READ, &hkey);

    if (hResult == ERROR_SUCCESS)
    {
        DWORD cbData = 0;
        hResult = RegGetValue(hkey, nullptr, valueName.c_str(), RRF_RT_REG_SZ, nullptr, nullptr, &cbData);

        if (hResult == ERROR_SUCCESS)
        {
            vector<BYTE> buffer(cbData + 1);

            cbData = (DWORD)buffer.size();
            hResult = RegGetValue(hkey, nullptr, valueName.c_str(), RRF_RT_REG_SZ, nullptr, &buffer[0], &cbData);

            wstring valueString(reinterpret_cast<wchar_t*>(&buffer[0]));

            found = valueString.find(L"Notepad++") != wstring::npos;
        }
    }

    RegCloseKey(hkey);

    if (found)
    {
        result = RegDeleteTree(HKEY_LOCAL_MACHINE, keyName.c_str());
    }

    return result;
}
```

It is being called from `NppShell::Installer::UnregisterOldContextMenu()`